arroyo/processing/processor.py

```diff
@@ -408,6 +417,12 @@ def _run_once(self) -> None:
                 self.__metrics_buffer.incr_timing(
                     "arroyo.consumer.poll.time", time.time() - start_poll
                 )
+
+                if self.__message is None and not self.__is_paused:
+                    if time.time() - self.__last_empty_msg_ts >= LOGGING_FREQUENCY_SEC:
+                        logger.info("Consumer is not paused, but did not receive a message from underlying consumer")
+                        self.__last_empty_msg_ts = time.time()
```

Comment:
I don't think this is right. 
Let's say we get None, then 30 seconds later we get a valid message, we are not resetting the counter. After 60 seconds we would log the message even if we received a message in the 60 seconds.
The goal was to periodically log if we did not received any message for a long period of time.